package.json

```
@@ -114,7 +114,7 @@
     "apollo-link-state": "^0.4.1",
     "config": "^2.0.1",
     "date-fns": "^1.29.0",
-    "formik": "^2.0.0",
+    "formik": "^1.5.7",
```

Should be be strict with `formik`'s version?
```suggestion
    "formik": "1.5.7",
```